index.js

```diff
@@ -1,65 +1,16 @@
-const chalk = require('chalk')
+
 const parse = require('ndjson').parse
 const through = require('through2').obj
 const { prettyFactory } = require('pino-pretty')
-const { prettifyTime } = require('pino-pretty/lib/utils')
-const prettyMs = require('pretty-ms')
-
-/**
- * @typedef {Object} HttpPrintOptions
- * @property {boolean} [all] support all log messages, not just HTTP logs
- * @property {boolean} [colorize] colorize logs, default is automatically set by chalk.supportsColor
- * @property {boolean|string} [translateTime] (default: false) When `true` the timestamp will be prettified into a string,
- *  When false the epoch time will be printed, other valid options are same as for `pino-pretty`
- * @property {boolean} [relativeUrl] (default: false)
- * @property {boolean} [lax] (default: false) When `true` the JSON parser will silently discard unparseable logs, e.g.
- * from nodemon
- */
```

Comment:
Why is this removed? It's still referenced below.

Reply:
I moved in the `lib/utils`

Should I replicate it?

Reply:
Yes. Please keep documentation where it exists. We should be adding more, not removing it.

Reply:
Done